### DISCUSSION TITLE
Tabs being treated as windows

### DISCUSSION THREAD

**Discussion by linuxdotexe** [zen-browser/desktop#729] [Q&A] [answered]
I just started using Zen a few days ago. I noticed that when I have multiple tabs open in the same window, my OS (Windows 10) treats these tabs as windows.
![image](https://github.com/user-attachments/assets/73981b22-19dc-4996-94fe-620e1ee83d2b)
![image](https://github.com/user-attachments/assets/296563bf-6443-4734-9414-88caf52ce95a)
I want to know if this is intended behavior and if I can turn if off. I am used to switching windows with the `Win + num` shortcut and this is causing a mild interruption to my flow. Please let me know what I can do to turn if off.

**Comment by FabiKel** [comment, marked as answer]
Hi, please check in the Zen settings under "General" if "Show tab previews in the Windows taskbar" is checked and disable it.

![grafik](https://github.com/user-attachments/assets/f3ae199d-e8de-48f5-a0e2-4ddfdeddc05a)

**Reply by linuxdotexe** [reply]
Thank you!
